spec-main/api-safe-storage-spec.ts

```diff
@@ -0,0 +1,102 @@
+import * as cp from 'child_process';
+import * as path from 'path';
+import { safeStorage } from 'electron/main';
+import { expect } from 'chai';
+import { emittedOnce } from './events-helpers';
+import { ifdescribe } from './spec-helpers';
+import * as fs from 'fs';
+
+/* isEncryptionAvailable returns false in Linux when running CI due to a mocked dbus. This stops
+* Chrome from reaching the system's keyring or libsecret. When running the tests with config.store
+* set to basic-text, a nullptr is returned from chromium,  defaulting the available encryption to false.
+*
+* Because all encryption methods are gated by isEncryptionAvailable, the methods will never return the correct values
+* when run on CI and linux.
+*/
+
+ifdescribe(process.platform !== 'linux')('safeStorage module', () => {
```

Comment:
@MarshallOfSound

I spent ~2 days trying to get the mocked dbus for networkmanager to work with isEncryptionAvailable for Linux. However, the mock isn't working consistently - would it be alright if we marked the tests as skipped in CI on Linux, and I filed a follow-up bug in the Electron issue tracker to get the mock dbus working? The tests do pass locally on Linux; happy to add that note

Reply:
Sounds reasonable to me